.travis.yml

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) 2013-2020 Arm Limited. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the License); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+language: sh
+os: linux
+dist: focal
+
+matrix:
+  include:
+
+    # ARM_MUSCA_S1
+
+    - &mbed-compile-test
+      stage: "Compile"
+      name: "Compile PSA example - release (ARM_MUSCA_S1)"
+      env: TARGET_NAME=ARM_MUSCA_S1 PROFILE=release
+      language: python
+      python: 3.8
+      install:
+        # Install arm-none-eabi-gcc
+        - pushd /home/travis/build && mkdir arm-gcc && cd arm-gcc
+        - curl -L0 "https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2020q2/gcc-arm-none-eabi-9-2020-q2-update-x86_64-linux.tar.bz2?revision=05382cca-1721-44e1-ae19-1e7c3dc96118&la=en&hash=D7C9D18FCA2DD9F894FD9F3C3DC9228498FA281A" --output gcc-arm-none-eabi-9-2020-q2-update.tar.bz2
+        - tar xf gcc-arm-none-eabi-9-2020-q2-update.tar.bz2
+        - export PATH="$PATH:${PWD}/gcc-arm-none-eabi-9-2020-q2-update/bin"
+        - popd
+        - arm-none-eabi-gcc --version
+        # Fetch mbed-os: We use manual clone, with depth=1 and --single-branch to save time.
+        - git clone --depth=1 --single-branch https://github.com/ARMmbed/mbed-os.git;
+        # Install Mbed CLI and dependencies
+        - pip install --upgrade mbed-cli
+        - pip install -r mbed-os/requirements.txt
+      script:
+        - echo mbed compile -t GCC_ARM -m ${TARGET_NAME} --profile ${PROFILE}
+        - mbed compile -t GCC_ARM -m ${TARGET_NAME} --profile ${PROFILE}
+
+    - <<: *mbed-compile-test
```

Comment:
Recommend to change to `mbed-psa-compile-test`

Reply:
I'll change it. I used `mbed-compile-test` because all the other examples have "cmake-build-test" in Travis regardless of what they are building and was trying to follow the same: https://github.com/ARMmbed/mbed-os-example-blinky/blob/2a95fcd8dd9cdde303a3b8210afd3e93a0b9f09a/.travis.yml#L90
So `mbed-compile-test` meant the test for `mbed compile`

Reply:
Done